trunner/testcase.py

```diff
@@ -90,23 +90,21 @@ def exec_test(self, proc):
         try:
             # Wait for a prompt
             proc.expect_exact('(psh)% ')
+            if self.exec_cmd:
+                if self.use_sysexec:
+                    self.sysexec(proc)
+                else:
+                    self.exec(proc)
         except (TIMEOUT, EOF) as exc:
             msg = 'Waiting for psh prompt failed!\n'
```

Comment:
Now if execution fails we will get misleading error message. Change it to "exec failed" or similar.

Reply:
Thanks, I've missed it.